tests/unit/test_main.py

```diff
@@ -1678,7 +1689,7 @@ def test_link_from_dict_non_existent_intf(self):
             "endpoint_b": {"id": "b"}
         }
         with self.assertRaises(ValueError):
-            self.napp._uni_from_dict(link_dict)
+            self.napp._link_from_dict(link_dict)
```

Comment:
From the method name and context, I figured out this was wrong and fixed it. No impacts to the coverage.

Reply:
We should probably also have another similar test for UNIs that checks the behaviour if the interface doesn't exist, similar to what the old version was incidentally checking.

Reply:
Good catch, @Ktmi! thank you! I've added a new test on commit 9ec5359

Please let me know if thaat is okay for you

Reply:
Perfect!